code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -3815,20 +3815,3 @@ datum
 			alch_strength = 0.3
 			description = "A cocktail from the prohibition era, named after a popular expression."
 			reagent_state = LIQUID
-
-		fooddrink/alcoholic/methacola
-			name = "Methacola"
-			id = "methacola"
-			fluid_r = 173
-			fluid_g = 124
-			fluid_b = 76
-			description = "A potent cocktail of amphetamines, caffeine and corn syrup. If this doesn't kill you then I don't know what will."
```

Comment:
descriptions probably shouldn't be in first person, since this is a non sentient computer's readout

Reply:
oh wait this is a removal whoops, phone dark mode made that ambiguous